tests/deb/deb_tests.bzl

```diff
@@ -15,6 +15,25 @@
 
 load("@bazel_skylib//lib:unittest.bzl", "analysistest", "asserts")
 
+def assert_contains(env, expected, got, msg = None):
+    """Asserts that the given `expected` occurs in a set of things.
+
+    Args:
+      env: The test environment returned by `unittest.begin`.
+      expected: An expected value.
+      go: The actual set returned by some computation.
+      msg: An optional message that will be printed that describes the failure.
+          If omitted, a default will be used.
+    """
+    if expected in got:
+        return
+    expectation_msg = "Expected %s in (%s)" % (expected, got)
+    if msg:
+        full_msg = "%s (%s)" % (msg, expectation_msg)
+    else:
+        full_msg = expectation_msg
+    fail(env, full_msg)
```

Comment:
How does all this look using something like [asserts.true](https://github.com/bazelbuild/bazel-skylib/blob/1.0.3/docs/unittest_doc.md#assertstrue)?

If that doesn't look good, then perhaps this at the bottom should be `unittest.fail()` or `analysistest.fail()` instead?  Documentation in [here](https://github.com/bazelbuild/bazel-skylib/blob/1.0.3/docs/unittest_doc.md).

Reply:
swtiched to analysistest.fail